llvm/test/CodeGen/RISCV/rvv/fold-scalar-load-crash.ll

```diff
@@ -12,21 +12,19 @@ define i32 @test(i32 %size, ptr %add.ptr, i64 %const) {
 ; RV32-NEXT:    vsetivli zero, 8, e8, mf2, ta, ma
```

Comment:
Have we lost the intention of this test?

Reply:
At least it doesn't crash? :-)
I will do some modifications to keep the loop.